src/library/playlisttablemodel.h

```diff
@@ -40,4 +40,5 @@ class PlaylistTableModel final : public TrackSetTableModel {
 
     int m_iPlaylistId;
     bool m_keepDeletedTracks;
+    QHash<int, QString> m_searchTexts;
```

Comment:
I guess this could be moved to [TrackSetmodel](**https://github.com/mixxxdj/mixxx/blob/main/src/library/trackset/tracksettablemodel.h**) but I didn't dare to touch that minimal class..